master/internal/sproto/resources.go

```diff
@@ -73,6 +73,8 @@ type ResourcesStarted struct {
 	Addresses []cproto.Address
 	// NativeResourcesID is the native Docker hex container ID of the Determined container.
 	NativeResourcesID string
+	// HPCJobID is the Slurm/PBS job ID.
+	HPCJobID string
```

Comment:
Sorry I missed this.. NativeResourcesID is exactly for something like HPCJobID. Can we just use it?

Reply:
It is intended to be the underlying RM id for the resources, so pod id, docker id or slurm job id

Reply:
as a general rule of thumb, as much as we can, slurm/hpc specific details shouldn't leak out of the RM